src/init.cc

```diff
@@ -976,7 +984,16 @@ ncclResult_t ncclCommInitRank(ncclComm_t* newcomm, int nranks, ncclUniqueId comm
   NVTX3_FUNC_RANGE_IN(nccl_domain);
   int cudaDev;
   CUDACHECK(cudaGetDevice(&cudaDev));
-  NCCLCHECK(ncclCommInitRankDev(newcomm, nranks, commId, myrank, cudaDev));
+  NCCLCHECK(ncclCommInitRankDev(newcomm, nranks, commId, myrank, cudaDev, nullptr));
+  return ncclSuccess;
+}
+
+NCCL_API(ncclResult_t, ncclCommInitRankWithScclXML, ncclComm_t* newcomm, int nranks, ncclUniqueId commId, int myrank, const char* xmlFile);
+ncclResult_t ncclCommInitRankWithScclXML(ncclComm_t* newcomm, int nranks, ncclUniqueId commId, int myrank, const char* xmlFile) {
```

Comment:
Please add a comment for each newly introduced function

Reply:
They are added in nccl.h.in for the API functions.